app/_src/gateway/reference/brotli.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Brotli compression
```

Comment:
```suggestion
title: Improve performance with Brotli compression
```
Just to give users an indication of what Brotli compression can do for them here